androidApp/src/main/java/dev/johnoreilly/confetti/venue/VenueView.kt

```diff
@@ -66,7 +70,27 @@ fun VenueView(venue: GetVenueQuery.Venue) {
     ) {
         Text(venue.name, style = MaterialTheme.typography.titleLarge)
         Spacer(modifier = Modifier.height(8.dp))
-        Text(venue.address.toString(), style = MaterialTheme.typography.titleSmall)
+        val mapLink = venue.mapLink
+        val address = venue.address
+        if (mapLink != null && address != null) {
+            ClickableText(
+                text = AnnotatedString(
+                    text = address,
+                    spanStyle = SpanStyle(
+                        color = MaterialTheme.colorScheme.onBackground,
```

Comment:
Somehow, setting the `style = MaterialTheme.typography.titleSmall` don't do anything, so I had to set the color _manually_